collaboration-service/index.js

```diff
@@ -30,6 +30,9 @@ io.on("connection", (socket) => {
 
     // Client joins a specific room.
     socket.on("join", ({ room }) => {
+        // if (io.sockets.adapter.rooms.get(room).size === 1) {
+        //     socket.broadcast.to(`${room}`).emit("update")
+        // }
```

Comment:
Is this suppose to be commented? If not can put it in another PR

Reply:
LGTM if this issue is resolved